setup.cfg

```diff
@@ -24,10 +24,12 @@ zip_safe = True
 packages = find:
 
 install_requires =
-  colorama~=0.4.1
-  packaging~=19.2
-  requests~=2.22
-  setuptools~=41.4
+  aiodns~=2.0
+  aiohttp~=3.6
```

Comment:
aiohttp, aiohttp and cchardet can be combined by specifying `aiohttp[speedups]~=3.6`

Reply:
I prefer explicit dependencies over implicit dependencies. BTW, now I use [vpip](https://github.com/eight04/vpip) to manage them.